pkg/model/gcemodel/autoscalinggroup.go

```diff
@@ -89,8 +89,8 @@ func (b *AutoscalingGroupModelBuilder) buildInstanceTemplate(c *fi.ModelBuilderC
 				BootDiskSizeGB: i64(int64(volumeSize)),
 				BootDiskImage:  s(ig.Spec.Image),
 
-				// TODO: Support preemptible nodes?
-				Preemptible: fi.Bool(false),
+				Preemptible:          fi.Bool(strings.EqualFold(fi.StringValue(ig.Spec.GCPProvisioningModel), "spot")),
+				GCPProvisioningModel: ig.Spec.GCPProvisioningModel,
```

Comment:
This can be with any case because or my proposal to allow that. Probably would be best to step back on that idea and only allow `SPOT` and `STANDARD` as options. Would make things much simpler I think.

Reply:
Rollback to uppcased variants only?

Reply:
Yes please. Thanks!

Reply:
fixed